spec/models/ems_event_spec.rb

```diff
@@ -196,6 +198,24 @@
           expect(new_event.availability_zone_id).to eq @availability_zone.id
         end
       end
+
+      context "when two events have the same timestamp" do
+        context "when events belong to the same object" do
```

Comment:
Were you going to add another test here?  If not, we can probably collapse these two contexts into one.